src/main/xquery/xqdoc2openapi-lib.xqm

```diff
@@ -56,7 +56,7 @@ as map(*)
             }
         else (
             map {
-                "type": $type/text()
+                "type": if (fn:contains($type/text(), "xs:")) then fn:replace($type/text(), "xs:", "") else $type/text()
```

Comment:
Is it necessary to drop `xs:`? Can you provide an example or screenshot of how this was before/after?

Reply:
OpenAPI uses the JSON data types (string, number, boolean, array, object, null).  It will need even more cases on the type.  Please see [https://www.w3schools.com/js/js_json_datatypes.asp](https://www.w3schools.com/js/js_json_datatypes.asp)

Reply:
Ok, got it. Thanks!